### ISSUE TITLE
[Noetic] rtabmap fails to build on armhf

### ISSUE THREAD

**Issue by sloretz** [introlab/rtabmap#1037]
Hey @matlabbe,

It looks like `rtabmap` isn't building on some platforms:

* [Ubuntu Focal armhf](https://build.ros.org/view/Nbin_ufhf_uFhf/job/Nbin_ufhf_uFhf__rtabmap__ubuntu_focal_armhf__binary/)

I'm planning to add all chronically failing packages to per-platform blacklists later this week.
That just means the buildfarm won't try to build them every day.

When all issues causing the build failures are resolved, please make a PR to the `production` branch on [ros_buildfarm_config](https://github.com/ros-infrastructure/ros_buildfarm_config/tree/production/noetic) removing it from the `package_blacklist`:

* [release-armhf-build.yaml](https://github.com/ros-infrastructure/ros_buildfarm_config/blob/production/noetic/release-armhf-build.yaml)

**Comment by matlabbe** [comment]
Yeah, no problem. It is failing because a dependency (libpointmatcher) is not built on armhf. The correct fix would be to do a conditional `depends` in rtabmap's `package.xml` based on the platform, but I doudt it is possible to do.
